### PR TITLE
feat: freeze browser version to 112

### PR DESCRIPTION
**Pull request by SuZhou-Joe**
### Description

Github CI is using the latest chromium to run the Cypress test, which leads to forever hang in some test cases.
According to [#4241](https://github.com/opensearch-project/opensearch-build/issues/4241) , we will freeze the Chromium version to 112, exactly same as what Jenkins CI uses.

### Issues Resolved

[List any issues this PR will resolve]

### Check List

- [x] Commits are signed per the DCO using --signoff

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

File: .github/workflows/release-e2e-workflow-template.yml [hunk 1]
```diff
@@ -127,7 +139,11 @@ jobs:
       - run: npx cypress cache path
       - name: Cypress tests
         uses: cypress-io/github-action@v2
+        env:
+          # remove success video to save space and memory
+          CYPRESS_DELETE_SUCCESS_VIDEOS: true
```
Comment (peterzhuamazon) [inline .github/workflows/release-e2e-workflow-template.yml:144]: Instead of record then delete videos, just disable recording:
https://github.com/opensearch-project/opensearch-dashboards-functional-test/blob/main/integtest.sh#L219-L223

**Review by peterzhuamazon — APPROVED** [review]

File: .github/workflows/release-e2e-workflow-template.yml (hunk 1 above)
Comment (SuZhou-Joe) [reply .github/workflows/release-e2e-workflow-template.yml]: Yes, but we want to keep the video if the specs failed.

File: cypress/support/index.js [hunk 2]
```diff
@@ -19,7 +19,7 @@
 // ***********************************************************
 
 // Import commands.js using ES2015 syntax:
-import '../utils/commands';
+import { currentBackendEndpoint } from '../utils/commands';
```
Comment (ruanyl) [inline cypress/support/index.js:22]: Unrelated?

**Review by ruanyl — APPROVED** [review]

File: .github/workflows/release-e2e-workflow-template.yml (hunk 1 above)
Comment (SuZhou-Joe) [reply .github/workflows/release-e2e-workflow-template.yml]: And even if we downgrade chromium version to 112, the security test will still fail. I am trying to reduce the memory our test cases are using.

File: cypress/support/index.js (hunk 2 above)
Comment (SuZhou-Joe) [reply cypress/support/index.js:22]: There is a lint warning in this file that we import `'../utils/commands'` twice, so I combine the two import into one.

**Comment by SuZhou-Joe** [comment]
Convert to draft as it is still WIP.

File: .github/workflows/release-e2e-workflow-template.yml (hunk 1 above)
Comment (peterzhuamazon) [reply .github/workflows/release-e2e-workflow-template.yml]: Got it.
